packages/pebble-shared/svgs/added-details.svg

```diff
@@ -0,0 +1 @@
+<svg width="1000" height="1000" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M83.333 250h500v83.333h-500V250zm0 166.667h500V500h-500v-83.333zm0 166.666h333.333v83.334H83.333v-83.334zm583.75 125l-125.417-125-62.5 62.5 187.917 187.5 291.25-291.666-62.5-62.5-228.75 229.166z" fill="#000"/></svg>
```

Comment:
most svgs are missing `width` and `height` attrs

Reply:
added.